includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1991,31 +1991,28 @@ protected function wc_setup_activate_get_feature_list() {
 			&& isset( $ppec_settings['enabled'] ) && 'yes' === $ppec_settings['enabled'];
 		$features['payment'] = $stripe_enabled || $ppec_enabled;
 
-		$features['taxes'] = (bool) get_option( 'woocommerce_setup_automated_taxes', false );
-
-		$domestic_rates  = (bool) get_option( 'woocommerce_setup_domestic_live_rates_zone', false );
-		$intl_rates      = (bool) get_option( 'woocommerce_setup_intl_live_rates_zone', false );
-		$features['rates'] = $domestic_rates || $intl_rates;
+		$features['taxes']  = (bool) get_option( 'woocommerce_setup_automated_taxes', false );
+		$features['labels'] = ( 'US' === WC()->countries->get_base_country() );
```

Comment:
Should this reflect whether the option remained checked on the Shipping step?

Reply:
Yes, it should - I'll update.

Reply:
Added in b9b9f19.